src/actions/actions.js

```diff
@@ -0,0 +1,48 @@
+import { endPoint, actionTypes } from "../constants/constants";
+
+export const showResults = results => ({
+    type: actionTypes.SEARCH_MOVIES,
+    results
+});
+
+export const changeSearchOption = option => ({
+    type: actionTypes.SEARCH_BY,
+    option
+});
+
+export const changeSortOption = option => ({
+    type: actionTypes.SORT_BY,
+    option
+});
+
+export const getSelectedMovie = movie => ({
```

Comment:
I'm not sure you're getting a movie. You rather set it (to store I mean)

Reply:
Done